src/javascript/_common/base/client_base.js

```diff
@@ -202,6 +202,8 @@ const ClientBase = (() => {
     const shouldCompleteTax = () => isAccountOfType('financial') &&
         !/crs_tin_information/.test((State.getResponse('get_account_status') || {}).status);
 
+    const isAuthenticationAllowed = () => !/allow_document_upload/.test((State.getResponse('get_account_status') || {}).status);
```

Comment:
state.getresponse handles not having the response itself, you can check its implementation throughout the codebase

```suggestion
    const isAuthenticationAllowed = () => !/allow_document_upload/.test(State.getResponse('get_account_status.status');
```